.github/workflows/tld-update.yml

```diff
@@ -39,7 +39,18 @@ jobs:
           committer: "GitHub <noreply@github.com>"
           author: "GitHub <noreply@github.com>"
           branch: psl-gtld-update
+          labels: |
+            ✅ autopull
+            🚩ICANN (IANA/ICP-3) Section
           delete-branch: true
+          
+      - name: Create or Update to List Add/Mod/Del To-Do Card
+        if: ${{ steps.cpr.outputs.pull-request-number }}
+        uses: peter-evans/create-or-update-project-card@v2
+        with:
+          project-number: 2
+          column-number: 8657926
```

Comment:
Should this be a string-typed `column-name` input instead? I don't see `column-number` listed in [the documented action inputs](https://github.com/peter-evans/create-or-update-project-card#action-inputs)